lib/jit/generation/elementwise_2d.cpp

```diff
@@ -117,6 +117,7 @@ void elementwise_2d::enqueue(driver::CommandQueue & /*queue*/, driver::Program c
   symbolic::set_arguments(expressions, kernel, current_arg);
 
   control.execution_options().enqueue(program.context(), kernel, global, local);
+  driver::dispatch::clFinish(control.execution_options().queue(expressions.context()).handle().cl());
```

Comment:
that's because the previous bug introduced two queues, this will temp fixed the bug on our test-suite, will test with the new upstream code then remove it